docker-compose.yml

```diff
@@ -1,6 +1,36 @@
-version: '3'
+version: "3"
```

Comment:
If we're on version 3, I'd consider using [the latest version in the series](https://docs.docker.com/compose/compose-file/).